### PR TITLE
[docs][branch] Document branch

### PR DESCRIPTION
**Pull request by schnappi17**
Document data branch and flink procudures.

### CODE REVIEW

**Comment by schnappi17** [comment]
@JingsongLi @FangYongs Please help to take a look at, thanks~

File: docs/content/maintenance/manage-branches.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+INSERT INTO t /*+ OPTIONS('branch' = 'branch1') */ SELECT ...
+```
+
+{{< /tab >}}
+
+{{< /tabs >}}
+
+## Merge Branch
+
+Merging the custom branch to main will delete all the snapshots, tags and schemas in the main branch that are created after the branch's initial tag. And copy snapshots, tags and schemas from the branch to the main branch.
```
Comment (JingsongLi) [inline docs/content/maintenance/manage-branches.md:98]: Can you explain more about the difference between merge and replace?

Comment (schnappi17) [reply docs/content/maintenance/manage-branches.md:98]: > Can you explain more about the difference between merge and replace?

Hi @JingsongLi, please help to review the update~

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**